scripts/publish.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-PACKAGES=("app" "bot-layer" "cdk" "cli" "core" "definitions" "fhir-router" "fhirtypes" "mock" "react")
+PACKAGES=("app" "bot-layer" "cdk" "cli" "core" "definitions" "eslint-config" "fhir-router" "fhirtypes" "mock" "react")
```

Comment:
This will publish `@medplum/eslint-config` in the next release.  That will be nice, because if anyone forks `foomedical`, they'll get the config.